<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton {border: none; 
text-align: left;
	font: 33pt &quot;Ubuntu&quot;;
color: white;

}

QPushButton:hover {
color: orange
}</string>
  </property>
  <widget class="QFrame" name="frHome">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>750</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>750</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>750</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frHome{background-image: url(:/home/assets/home.jpg)}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>201</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnPlay">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none</string>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHistory">
       <property name="text">
        <string>History</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOptions">
       <property name="text">
        <string>Options</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogout">
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frLogin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>125</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {background-color: #302E2B;
border: 10px solid #FFB12C;}

QPushButton:hover {
text-decoration: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnToRegister">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>369</y>
       <width>163</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color: #EE4612;
text-align: center;
font-size: 16px;
font-weight: bold;}


QPushButton:hover{background-color: rgba(238, 70, 18, 0.7);color:white}</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUsername">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>155</y>
       <width>409</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background-color: rgba(255,255,255,0.5);
color: black;s</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPassword">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>220</y>
       <width>409</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background-color: rgba(255,255,255,0.5);
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbUsername">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>155</y>
       <width>409</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
color: black</string>
     </property>
     <property name="text">
      <string>Username...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogin">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>286</y>
       <width>409</width>
       <height>54</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color: #F59701;
text-align: center;
font-size: 22px;
font-weight: bold;}
QPushButton:hover{background-color: rgba(238, 70, 18, 0.7); color:white}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>377</y>
       <width>230</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none; color: white;</string>
     </property>
     <property name="text">
      <string>New? Sign up right now!</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbLogo">
     <property name="geometry">
      <rect>
       <x>138</x>
       <y>18</y>
       <width>224</width>
       <height>108</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbPassword">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>220</y>
       <width>409</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
color: black</string>
     </property>
     <property name="text">
      <string>Password...</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbNoti">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>430</y>
       <width>409</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRegister">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>286</y>
       <width>409</width>
       <height>54</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color: #EE4612;
text-align: center;
font-size: 22px;
font-weight: bold;}


QPushButton:hover{background-color: rgba(238, 70, 18, 0.7);color:white}</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnToLogin">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>369</y>
       <width>163</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color: #F59701;
text-align: center;
font-size: 16px;
font-weight: bold;}
QPushButton:hover{background-color: rgba(238, 70, 18, 0.7); color:white}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQuit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>lbUsername</zorder>
    <zorder>lbPassword</zorder>
    <zorder>btnLogin</zorder>
    <zorder>label_2</zorder>
    <zorder>btnToRegister</zorder>
    <zorder>txtPassword</zorder>
    <zorder>txtUsername</zorder>
    <zorder>lbNoti</zorder>
    <zorder>btnRegister</zorder>
    <zorder>btnToLogin</zorder>
    <zorder>btnQuit</zorder>
    <zorder>lbLogo</zorder>
   </widget>
   <widget class="QLabel" name="lbDim">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>750</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0.7)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>lbDim</zorder>
   <zorder>frLogin</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
